test/e2e/kubectl.go

```diff
@@ -566,8 +566,15 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 			framework.SkipUnlessProviderIs("gke")
 			nsFlag := fmt.Sprintf("--namespace=%v", ns)
 			podJson := readTestFileOrDie(kubectlInPodFilename)
+
+			// Replace the host path to kubectl in json
+			podString := strings.Replace(string(podJson),
+				"$KUBECTL_PATH",
+				framework.TestContext.KubectlPath,
```

Comment:
@dims  If flag [kubectl-path](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/test_context.go#L184) is not set, `framework.TestContext.KubectlPath` may be an empty string, right?